_data/tools.yml

```diff
@@ -1885,3 +1885,17 @@
   v2: true
   v3: true
   v3_1: true
+   
+- name: BlocklyAutomation
+  category: 
+    - code-generators
+    - sdk
+    - documentation
+    - testing
+  link: https://ignatandrei.github.io/BlocklyAutomation/
+  github: http://github.com/ignatandrei/BlocklyAutomation
+  language: Javascript / .NET
+  description:
+    Input any OpenAPI definition file in order to have generated Blocks in Blockly form to test and generate documentation    
```

Comment:
```suggestion
    Input any OpenAPI document to have generated Blocks in Blockly form to test and generate documentation.
```

Reply:
Also what is a block? This feels like we are defining something with the word, instead of explaining what it does.